dfastmi/io/Branch.py

```diff
@@ -0,0 +1,14 @@
+from dfastmi.io.Reach import Reach
+
+
+from typing import List
+
+
+class Branch():
```

Comment:
don't forget documentation.

Reply:
will add

Reply:
Discussed, we will leave these comments for a later refactoring.
This will be tackled in https://issuetracker.deltares.nl/browse/DFAST-112

Reply:
will be done in batch refactoring issue